contribs/gmf/src/datasource/OGC.js

```diff
@@ -24,6 +24,11 @@ const exports = class extends ngeoDatasourceOGC {
      */
     this.gmfLayer_ = options.gmfLayer;
 
+    /**
+     * @type {?Array.<string>}
+     * @export
+     */
+    this.columnsOrder_ = options.columnsOrder;
```

Comment:
In a datasource, we the term "column" is not used.  We use "attribute".  I think it would make more sense to use `attributesOrder` here.

Also, the type shold be `Array.<string>|undefined`.

Also, we're missing the type definition of of the new option in `ngeox.datasource.DataSourceOptions`